packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -116,11 +119,11 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
         </div>
       </div>
     </div>
-    <div class="bg-charcoal-600 mt-5 rounded-md p-3">
+    <div class="bg-charcoal-600 mt-5 rounded-md p-3" role="region" aria-label="InstalledExtensions">
       <table class="min-w-full">
         <tbody>
           {#each $extensionInfos as extension}
-            <tr class="border-y border-gray-900">
+            <tr class="border-y border-gray-900" aria-label="{extension.name}">
               <td class="px-6 py-2">
```

Comment:
Here I can imagine `aria-label="Extension state or information"`

Reply:
I wanted to be able to easily reference this element by using exact name of the extension as a label to search for this element

Reply:
done.